packages/core/src/crawlers/statistics.ts

```diff
@@ -159,17 +159,15 @@ export class Statistics {
         this._teardown();
     }
 
-    async resetStore() {
-        if (!this.enablePersistence) {
+    async resetStore(opts?: {
+        /**
+         * If true, manually reset KV store entry for the statistics, ignoring the {@apilink StatisticsOptions.enablePersistence} flag
```

Comment:
I am not sure about the word "manually" here. If you call this method, its always manual usage, regardless of the options.